cmd/clairctl/docker/login.go

```diff
@@ -0,0 +1,47 @@
+package docker
```

Comment:
Looks like there's a looot of Docker related code. I bet we could simplify everything by simply using the various Docker libraries. Right?